config/sync/views.settings.yml

```diff
@@ -0,0 +1,46 @@
+display_extenders: {  }
```

Comment:
Is this config related to the `views_ui` module? If yes, it must be removed.

Reply:
As mentioned in description this file is needed for views ui to work.
It is related to the views module which is already activated.
web/core/modules/views/config/install/views.settings.yml 
So we miss it when enabling views_ui